src/ipa_pc_de/mod.rs

```diff
@@ -684,7 +684,7 @@ impl<G: AffineCurve, D: Digest> PolynomialCommitment<G> for InnerProductArgPC<G,
         })
     }
 
-    /// The verification function of an opening proof produced by ``open_individual_opening_challenges()``
+    /// The verification function of an opening proof produced by `fn open()`
     fn verify<'a>(
```

Comment:
Referring to the lines below: we should add an inline comment in which we specify that "check_poly = h(X) = prod (1 + xi_{log(d+1) - i} * X^{2^i} )".
Is check_poly the most appropriate name for that variable? I would call h_poly instead.